build.gradle

```diff
@@ -277,8 +277,8 @@ dependencies {
     compile 'org.jooq:jool-java-8:0.9.14'
     compile 'com.github.hmcts:ccd-case-document-am-client:1.7.1'
 
-    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.3'
-    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.2-Categories'
+    testCompile group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.5_HMAN-368'
+    testCompile group: 'com.github.hmcts', name: 'befta-fw', version: '8.7.6-HMAN-368'
```

Comment:
These should be updated to match final version numbers if being merged to `master`.